context/app/static/js/components/detailPage/Attribution/Attribution.jsx

```diff
@@ -1,11 +1,15 @@
 import React from 'react';
 import SectionHeader from 'js/shared-styles/sections/SectionHeader';
+import { useFlaskDataContext } from 'js/components/Contexts';
 import { DetailPageSection } from 'js/components/detailPage/style';
 import EmailIconLink from 'js/shared-styles/Links/iconLinks/EmailIconLink';
 import { FlexPaper } from './style';
 import SectionItem from '../SectionItem';
 
-function Attribution({ group_name, created_by_user_displayname, created_by_user_email }) {
+function Attribution() {
```

Comment:
This `Attribution` component is also used on the Donor, Dataset, and Preview pages (last 3 links under `Resources` in the header) - does this work correctly on all of them?

Reply:
Yeah, they are all coming from flaskdata context, so they are working properly. Double checked again to make sure.